lib/rules/attribute-order.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @fileoverview enforce ordering of attributes
+ * @author Erin Depew
+ */
+'use strict'
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+function getPosition (attribute, attributeOrder) {
+  const name = attribute.key.name
+  if (name === 'is') {
+    return attributeOrder.indexOf('DEFINITION')
+  } else if (name === 'for') {
+    return attributeOrder.indexOf('LIST_RENDERING')
+  } else if (name === 'if' || name === 'else-if' || name === 'else' || name === 'show' || name === 'cloak') {
+    return attributeOrder.indexOf('CONDITIONALS')
+  } else if (name === 'pre' || name === 'once') {
+    return attributeOrder.indexOf('RENDER_MODIFIERS')
+  } else if (name === 'id') {
+    return attributeOrder.indexOf('GLOBAL')
+  } else if (name === 'ref' || name === 'key' || name === 'slot') {
+    return attributeOrder.indexOf('UNIQUE')
+  } else if (name === 'model') {
+    return attributeOrder.indexOf('BINDING')
+  } else if (name === 'on') {
+    return attributeOrder.indexOf('EVENTS')
```

Comment:
Does it catch both `v-on` and `@`?

Reply:
yup! I'm using `@click` in my tests and it seems to work fine, I'll also add `v-on` just to be sure